src/rgw/rgw_object_expirer_core.cc

```diff
@@ -52,11 +52,14 @@ int RGWObjectExpirer::init_bucket_info(const string& tenant_name,
    * punching the tenant through the objexp_hint_entry, but now we
    * find that our instances do not actually have tenants. They are
    * unique thanks to IDs. So the tenant string is not needed...
+
+   * XXX reloaded: it turns out tenants were needed after all since bucket ids
```

Comment:
What does XXX mean?

Reply:
there was XXX above in teh comment, this was just meant to be a pun movie reference, can drop that